guac-install.sh

```diff
@@ -568,6 +642,9 @@ echo
 # Done
 echo -e "${BLUE}Installation Complete\n- Visit: http://localhost:8080/guacamole/\n- Default login (username/password): guacadmin/guacadmin\n***Be sure to change the password***.${NC}"
 
-if [ "$installDuo" = true ]; then
+if [ "${installDuo}" = true ]; then
     echo -e "${YELLOW}\nDon't forget to configure Duo in guacamole.properties. You will not be able to login otherwise.\nhttps://guacamole.apache.org/doc/${GUACVERSION}/gug/duo-auth.html${NC}"
 fi
+
+sleep 2s
+xdg-open http://localhost:8080/guacamole/
```

Comment:
Not sure what this is? Is it trying to open the URL from the system? I imagine most people run this headless.

Reply:
Correct
And most people also don't use root as default - so it wouldn't work